Source/Editor/Windows/Filters/LevelRangeFilterWindow.cs

```diff
@@ -27,11 +30,8 @@ protected override void Configure()
 
         protected override void Load(XmlNode filterNode)
         {
-            string levelMin = filterNode.SelectSingleNode("levelMin")?.Attributes?["value"]?.Value;
-            xMinLevelComboBox.SelectedItem = levelMin;
-
-            string levelMax = filterNode.SelectSingleNode("levelMax")?.Attributes?["value"]?.Value;
-            xMaxLevelComboBox.SelectedItem = levelMax;
+            xMinLevelComboBox.SelectedItem = filterNode.GetValueAttributeValueFromChildElement(LevelMinName);
+            xMaxLevelComboBox.SelectedItem = filterNode.GetValueAttributeValueFromChildElement(LevelMaxName);

```

Comment:
What happens if the value read from the configuration doesn't match any of the existing combo box values?

Reply:
SelectedItem will be null. Probably fine for now.